src/constants.ts

```diff
@@ -5,3 +5,5 @@ export const DEFAULT_STATIC_MAX_AGE = Duration.days(30).toSeconds();
 export const DEFAULT_STATIC_STALE_WHILE_REVALIDATE = Duration.days(1).toSeconds();
 
 export const LAMBDA_RUNTIME = Runtime.NODEJS_18_X;
+
+export const DEFAULT_LAMBA_MEMORY = 1536;
```

Comment:
Linking to the article would be useful for future reference of this magic number

Reply:
tl;dr: 1536mb costs 1.5x but runs twice as fast for most scenarios.